Sources/MapboxMaps/Gestures/GestureHandlers/SingleTapGestureHandler.swift

```diff
@@ -25,3 +26,9 @@ internal final class SingleTapGestureHandler: GestureHandler {
         }
     }
 }
+
+extension SingleTapGestureHandler: UIGestureRecognizerDelegate {
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return gestureRecognizer is UITapGestureRecognizer
```

Comment:
should we also check condition of `otherGestureRecognizer` being of type `UITapGestureRecognizer`?
and maybe a check `gestureRecognizer === self.gestureRecognizer` is better here and more consistent to other GestureHandler i.e https://github.com/mapbox/mapbox-maps-ios/blob/main/Sources/MapboxMaps/Gestures/GestureHandlers/PinchGestureHandler.swift#L124